tests/data/split_merge/.trestle/config.ini

```diff
@@ -5,29 +5,29 @@ targets = trestle.oscal.target.Target
 target-control-implementations = trestle.oscal.target.TargetControlImplementation
 
 [catalog]
-decomposition = ['groups[].controls[]']
+decomposition_rules = ['catalog.groups.*.controls.*']
 number_of_groups = 2
```

Comment:
I'm not sure I understand the enumerations here by the names. I think you need to change the variable names to be explicitly 'init' based. E.g. haow would these variables be used by different commands? (or confuse the users because of it).

Reply:
The default decomposition rules would be used in `trestle create` and `trestle import` commands only.
We could potentially use them as well in `trestle split` if no arguments are passed to a `trestle split` command and if it is run from the base of a model project. But I would avoid that because of the number of 
if's which can cause more confusion to the user.